src/Provider/ExecutionProvider.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Digia\GraphQL\Provider;
+
+use Digia\GraphQL\Execution\Execution;
+use Digia\GraphQL\Execution\ExecutionInterface;
+use League\Container\ServiceProvider\AbstractServiceProvider;
+
+/**
+ * Class ExecutionProvider
+ * @package Digia\GraphQL\Provider
+ */
+class ExecutionProvider extends AbstractServiceProvider
+{
+    /**
+     * @var array
+     */
+    protected $provides = [
+        ExecutionInterface::class
+    ];
+
+    /**
+     * @inheritdoc
+     */
+    public function register()
+    {
+        $this->container->add(ExecutionInterface::class, function () {
```

Comment:
Please change the second argument to `Execution::class`. Also, should this be a shared instance? If so, give `true` as the third argument.